### PR TITLE
Direct Access to Monitoring Service API

### PR DESCRIPTION
**Pull request by raptorsun**
Mention Openshift provides direct access to API endpoints in 3rd party monitoring components.

### CODE REVIEW

File: monitoring/accessing-third-party-service-apis.adoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
+[id="accessing-third-party-service-apis"]
+= Accessing third-party Service APIs
+include::modules/common-attributes.adoc[]
+:context: accessing-third-party-service-apis
+
+toc::[]
+
+{product-title} provides direct access to third-party web service APIs for components in the Monitoring stack, such as Prometheus, Alertmanager, or Thanos Querier, using a Bearer Token for authentification.
```
Comment (juzhao) [inline monitoring/accessing-third-party-service-apis.adoc:8]: need to add Thanos Ruler

Comment (juzhao) [inline monitoring/accessing-third-party-service-apis.adoc:8]: normal users with their  Bearer Token won't be able to get the alertmanager-main route, should clarify the least required permission

**Comment by bburt-rh** [comment]
@raptorsun @juzhao Sorry, but I ended up submitting a separate PR for this content to https://github.com/openshift/openshift-docs/pull/41384. So you can close this PR and work on my new one instead. Thanks!

**Comment by juzhao** [comment]
based on previous comment from @bburt-rh , close this PR

**Pull request closed without merging**